GPU/Vulkan/PipelineManagerVulkan.cpp

```diff
@@ -31,8 +31,11 @@ void PipelineManagerVulkan::Clear() {
 	// store the keys.
 
 	pipelines_.Iterate([&](const VulkanPipelineKey &key, VulkanPipeline *value) {
-		if (value->pipeline)
-			vulkan_->Delete().QueueDeletePipeline(value->pipeline);
+		if (value->pipeline) {
+			VkPipeline pipeline = value->pipeline->pipeline;
+			vulkan_->Delete().QueueDeletePipeline(pipeline);
```

Comment:
Hm, I wonder if we need to wait if it is not set yet, in case we try to delete it while compiling or something?

-[Unknown]

Reply:
Yeah, I somehow didn't think of that case, needs checks indeed.